from afs.exceptions.AfsError import AfsError

class FServError(AfsError):
    pass
